app/models/order.rb

```diff
@@ -0,0 +1,4 @@
+class Order < ApplicationRecord
+	has_many :order_items
+	has_many :products, through: :order_items
```

Comment:
@Anuradha-yadav169, we should also create an association between order and user. Please make the necessary changes.